src/main/java/com/likelion/backendplus4/talkpick/backend/auth/presentation/support/validator/SignUpDtoValidator.java

```diff
@@ -0,0 +1,383 @@
+	 * @since 2025-05-14
+	 * @modified 2025-05-14
+	 */
+	private boolean isNameEmpty(String name, Errors errors) {
+		if (!StringUtils.hasText(name)) {
+			AuthValidationError.NAME_EMPTY.reject(errors);
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * 이름에 공백이 포함되었는지 확인하고, 포함된 경우 오류를 등록합니다.
+	 *
+	 * @param name   검증할 이름 문자열
+	 * @param errors Errors 객체
+	 * @return 공백이 포함된 경우 true, 아니면 false
+	 * @author 박찬병
+	 * @since 2025-05-14
+	 * @modified 2025-05-14
+	 */
+	private boolean hasNameWhitespace(String name, Errors errors) {
+		if (StringUtils.containsWhitespace(name)) {
+			AuthValidationError.NAME_WHITESPACE.reject(errors);
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * 이름의 길이가 30자를 초과하는지 확인하고, 초과 시 오류를 등록합니다.
+	 *
+	 * @param name   검증할 이름 문자열
+	 * @param errors Errors 객체
+	 * @author 박찬병
+	 * @since 2025-05-14
+	 * @modified 2025-05-14
+	 */
+	private void checkNameSize(String name, Errors errors) {
+		if (30 < name.length()) {
+			AuthValidationError.NAME_SIZE.reject(errors);
+		}
+	}
+
+	/**
+	 * 이름에 영문·한글 이외의 문자가 포함되었는지 검사하고, 포함 시 오류를 등록합니다.
+	 *
+	 * @param name   검증할 이름 문자열
+	 * @param errors Errors 객체
+	 * @author 박찬병
+	 * @since 2025-05-14
+	 * @modified 2025-05-14
+	 */
+	private void checkNamePattern(String name, Errors errors) {
+		if (!NAME_PATTERN.matcher(name).matches()) {
+			AuthValidationError.NAME_INVALID_CHAR.reject(errors);
+		}
+	}
+
+	/**
+	 * 닉네임이 null 또는 blank인지 확인하고, 빈 값일 경우 오류를 등록합니다.
+	 *
+	 * @param nick   검증할 닉네임 문자열
+	 * @param errors Errors 객체
+	 * @return 빈 값일 경우 true, 아닐 경우 false
+	 * @author 박찬병
+	 * @since 2025-05-14
+	 * @modified 2025-05-14
+	 */
+	private boolean isNickEmpty(String nick, Errors errors) {
+		if (!StringUtils.hasText(nick)) {
+			AuthValidationError.NICKNAME_EMPTY.reject(errors);
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * 닉네임에 공백이 포함되었는지 확인하고, 포함된 경우 오류를 등록합니다.
+	 *
+	 * @param nick   검증할 닉네임 문자열
+	 * @param errors Errors 객체
+	 * @return 공백이 포함된 경우 true, 아니면 false
+	 * @author 박찬병
+	 * @since 2025-05-14
+	 * @modified 2025-05-14
+	 */
+	private boolean hasNickWhitespace(String nick, Errors errors) {
+		if (StringUtils.containsWhitespace(nick)) {
+			AuthValidationError.NICKNAME_WHITESPACE.reject(errors);
+			return true;
+		}
+		return false;
+	}
+
+	/**
+	 * 닉네임의 길이가 20자를 초과하는지 확인하고, 초과 시 오류를 등록합니다.
+	 *
+	 * @param nick   검증할 닉네임 문자열
+	 * @param errors Errors 객체
+	 * @author 박찬병
+	 * @since 2025-05-14
+	 * @modified 2025-05-14
+	 */
+	private void checkNickSize(String nick, Errors errors) {
+		if (20 < nick.length()) {
+			AuthValidationError.NICKNAME_SIZE.reject(errors);
+		}
+	}
+
+	/**
+	 * 닉네임에 영문·한글 이외의 문자가 포함되었는지 검사하고, 포함 시 오류를 등록합니다.
+	 *
+	 * @param nick   검증할 이름 문자열
+	 * @param errors Errors 객체
+	 * @author 박찬병
+	 * @since 2025-05-14
+	 * @modified 2025-05-14
+	 */
+	private void checkNickNamePattern(String nick, Errors errors) {
+		if (!NICKNAME_PATTERN.matcher(nick).matches()) {
+			AuthValidationError.NAME_INVALID_CHAR.reject(errors);
+		}
+	}
```

Comment:
P4
AuthValidationError에 써있는 오류메세지를보면
주석과 일치하는 부분은 Name이아닌 NICKNAME_INVALID_CHAR 인거같습니다

```suggestion
	private void checkNickNamePattern(String nick, Errors errors) {
		if (!NICKNAME_PATTERN.matcher(nick).matches()) {
			AuthValidationError.NICKNAME_INVALID_CHAR.reject(errors);
		}
	}
```